doc/src/arch/reference.rst

```diff
@@ -826,6 +826,8 @@ The following tags are common to all ``<tile>`` tags:
 
 .. arch:tag:: <sub_tile name"string" capacity="{int}">
 
+    .. seealso:: For a tutorial on describing the usage of sub tiles for ``heterogeneous tiles`` definition see :ref:`heterogeneous_tiles_tutorial`.
```

Comment:
I feel like we need a definition of heterogeneous tiles here (novice users wont know what that means). Something along the lines of: heterogeneous tiles (tiles which support multiple instances of the same or different pb_types).